cross-chain/solana/migrations/deploy.ts

```diff
@@ -1,12 +1,78 @@
-// Migrations are an early feature. Currently, they're nothing more than this
-// single deploy script that's invoked from the CLI, injecting a provider
-// configured from the workspace's Anchor.toml.
-
-const anchor = require("@coral-xyz/anchor");
+import * as anchor from "@coral-xyz/anchor"
+import fs from "fs"
+import { PublicKey, Keypair } from "@solana/web3.js"
+import dotenv from "dotenv"
+import { PROGRAM_ID as METADATA_PROGRAM_ID } from "@metaplex-foundation/mpl-token-metadata";
 
 module.exports = async function (provider) {
-  // Configure client to use the provider.
-  anchor.setProvider(provider);
+  dotenv.config({ path: "../solana.env" })
+
+  anchor.setProvider(provider)
+
+  const tbtcProgram = anchor.workspace.Tbtc
+  // This wallet deployed the program and is also an authority
+  const authority = loadKey(process.env.AUTHORITY)
+  const newAuthority = process.env.THRESHOLD_COUNCIL_MULTISIG
```

Comment:
You may want to call your transfer authority script in this deploy script (or add the transfer authority instruction to the transaction after you intialize; you can put these two instructions in the same transaction because you're not sending that much data with either of these instructions).